tests/integration/lsp_tests.rs

```diff
@@ -12106,7 +12049,7 @@ fn lsp_deno_future_env_byonm() {
 }
 
 #[test]
-fn lsp_sloppy_imports_warn() {
```

Comment:
I removed the warning. It's not worth maintaining because it's a lot of work to add when this is done from the resolver. We can add this back as a lint rule later (see https://github.com/denoland/deno/issues/22844#issuecomment-1989103409)